sanic_routing/tree.py

```diff
@@ -17,7 +18,7 @@ def __init__(
         part: str = "",
         root: bool = False,
         parent: t.Optional["Node"] = None,
```

Comment:
Can take away the string annotation here too.

Reply:
I didn't know that annotations referencing themselves were also applied.

thanks for the good point